messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -864,6 +865,28 @@ public Throwable getError() {
             return errorState;
         }
 
+        /**
+         * Return the estimated relative replay position this Segment represents.
```

Comment:
Nit: You could use `{@inheritDoc}` to copy over the documentation from the interface so that you do not have to duplicate it.